boltdb/adaptor_test.go

```diff
@@ -0,0 +1,221 @@
+package boltdb_test
+
+import (
+	"context"
+	"errors"
+
+	"github.com/dogmatiq/dogma"
+	"github.com/dogmatiq/enginekit/fixtures"
+	. "github.com/dogmatiq/projectionkit/boltdb"
+	"github.com/dogmatiq/projectionkit/boltdb/internal"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"go.etcd.io/bbolt"
+)
+
+var _ = Describe("MessageHandler Adaptor", func() {
+	var (
+		mh *internal.MessageHandlerMock
+		db *internal.TempDB
+	)
+
+	BeforeEach(func() {
+		mh = &internal.MessageHandlerMock{}
+		mh.ConfigureCall = func(c dogma.ProjectionConfigurer) {
+			c.Identity("<projection>", "<key>")
+		}
+		db = internal.NewTempDB()
+	})
+
+	AfterEach(func() {
+		db.Close()
+	})
+
+	Context("HandleEvent method", func() {
+
+		It("executes as expected", func() {
```

Comment:
It's unclear what this test is checking for. Is it about checking that OCC is enforced (it probably should be considering that's what the adaptor does), and if so, can it be split up further?  (see that it's using `By()` but I'd still favour smaller tests *if* possible).

Reply:
As discussed, the tests structure remained the same to support the building preset for the subsequent tests, but all variables were replaced with literals to make the tests more explicit.